scripts/load-secrets.ps1

```
@@ -0,0 +1,59 @@
+foreach ($prop in $mapping.PSObject.Properties) {
+  $envKey = $prop.Name
+  $itemId = $prop.Value.item_id
+  $field = $prop.Value.field
+  if (-not $itemId -or $itemId -like 'REPLACE-WITH*') {
+    $placeholders += $envKey
+    continue
+  }
+  try {
+    $json = bw get item $itemId | ConvertFrom-Json
+  } catch {
+    Write-Warning "Failed to fetch item $itemId for $envKey: $_"
```

PSScriptAnalyzer (1.24.0)

[error] Variable reference is not valid. ':' was not followed by a valid variable name character. Consider using ${} to delimit the name.

(InvalidVariableReferenceWithDrive)